Content.Server/Medical/PenLightSystem.cs

```diff
@@ -0,0 +1,116 @@
+using Content.Server.DoAfter;
+using Content.Server.PowerCell;
+using Content.Shared.Damage;
+using Content.Shared.DoAfter;
+using Content.Shared.Drugs;
+using Content.Shared.Drunk;
+using Content.Shared.Eye.Blinding.Components;
+using Content.Shared.Interaction;
+using Content.Shared.Medical;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Traits.Assorted.Components;
+using Robust.Server.GameObjects;
+using Robust.Shared.Player;
+using Robust.Shared.Timing;
+
+namespace Content.Server.Medical;
+/// <summary>
+/// This stores the eye exam system for <see cref="PenLightComponent"/>
+/// </summary>
+public sealed class PenLightSystem : EntitySystem
+{
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+    [Dependency] private readonly DoAfterSystem _doAfter = default!;
+    [Dependency] private readonly PowerCellSystem _powerCell = default!;
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<PenLightComponent, AfterInteractEvent>(OnAfterInteract);
+        SubscribeLocalEvent<PenLightComponent, PenLightDoAfterEvent>(OnDoAfter);
+    }
+
+    private void OnAfterInteract(EntityUid uid, PenLightComponent component, AfterInteractEvent args)
+    {
+        if (args.Handled || args.Target is not { } target)
+            return;
+        args.Handled = TryStartExam(uid, target, args.User, component);
+    }
+
+    private void OnDoAfter(Entity<PenLightComponent> uid, ref PenLightDoAfterEvent args)
+    {
+        if (args.Handled || args.Cancelled || args.Target == null || !_powerCell.HasDrawCharge(uid, user: args.User))
+            return;
```

Comment:
```suggestion
    {
        if (args.Handled 
            || args.Cancelled 
            || args.Target == null 
            || !_powerCell.HasDrawCharge(uid, user: args.User))
            return;
```

Reply:
Easier to at a glance read the guard clauses if each OR Condition is on its own line.